Zelda/Zelda.cpp

```diff
@@ -1,1205 +0,0 @@
-			{
-				itemFound = true;
-
-				if (!isBagFull())
-				{
-					int j = 0;
-
-					for (; bag[j] != nullptr; ++j)
-					{
-
-					}
-
-					bag[j] =  (current_Room->getItemsPresent())[i];
-					current_Room->setItemsPresent(i , nullptr);
-
-					picked = true;
-
-					CashUpdater();
-				}
-				else
-					cout << "\nSorry , your bag is already full.";
-			}
-		}
-	}
-
-	HelperFunctions::color(RED);
-	if(picked)
-		cout <<endl << itemName << " has successfully been picked.";
-
-	else if(itemFound == false)
-		cout <<endl << itemName <<" is not present in this room.";
-}
-
-
-
-
-void Player::Drop(char const* itemName)
-{
-	bool dropped = false;
-	bool item_in_bag = false;
-
-	for (int i = 0; i < 10; ++i)
-	{
-		if(bag[i] != NULL)
-		{
-			if(	strcmp((bag[i])->getItemName() , itemName) == 0)
-			{
-				item_in_bag = true;
-
-				if(!(current_Room-> isRoomFull()))
-				{
-					int j = 0;
-
-					for (; (current_Room->getItemsPresent())[j] != nullptr; ++j)
-					{
-
-					}
-
-					current_Room->setItemsPresent(j , bag[i]);
-					bag[i] = nullptr;
-
-					dropped = true;
-
-					CashUpdater();
-				}
-				else
-				{
-					HelperFunctions::color(AQUA);
-					cout << "/n/The item" << itemName << " couldn't be dropped as Room "<<current_Room->getRoomNumber()
-						 << " is already full.";
-				}
-			}
-		}
-	}
-
-	HelperFunctions::color(YELLOW);
-
-	if(dropped)
-		cout << endl<< itemName <<" has successfully been dropped.";
-
-	else if(item_in_bag == false)
-		cout << endl << itemName <<" couldn't be dropped as it is not present in your bag.";
-}
-
-
-
-
-
-bool Player::Attack(char const* monsterName)
-{
-	bool killed = false;
-
-	if(current_Room->getMonsterPresent() != nullptr)
-	{
-		if(((current_Room->getMonsterPresent())->getLivingState()) == true)
-		{
-			for (int i = 0; i < 10; ++i)
-			{
-				if( (current_Room->getMonsterPresent())->getKillingWeapon() == bag[i] )
-				{
-						(current_Room->getMonsterPresent())->setLivingState(false);
-						HelperFunctions::color(RED);
-						cout << monsterName <<" has been killed . ";
-
-						killed = true;
-				}
-			}
-
-			if (killed == false)
-			{
-				setCurrentState(false);
-				HelperFunctions::color(RED);
-				cout << "/nYour bag doesn't contain the Weapon required to kill " << monsterName;
-			}
-		}
-
-		else if ((((current_Room->getMonsterPresent())->getLivingState()) == false))
-			cout << endl << monsterName << " is already dead.";
-
-	}
-
-	else
-		cout << endl << monsterName <<" is not present in this room.";
-
-	return killed;
-}
-
-
-
-void Player::Look()
-{
-	HelperFunctions::color(RED);
-	cout << "\nCurrently you are in Room "<<current_Room->getRoomNumber()<<". ";
-	cout << current_Room->getDescription();
-
-	for (int i = 0; i < 4; ++i)
-	{
-		HelperFunctions::color(AQUA);
-		if ((current_Room->getPaths())[i] != NULL)
-			cout << " There is a room to your " << current_Room->directionName(i)<<" .";
```

Comment:
Do not include a space before the period.

Reply:
really?